repository-data/webfiles/src/main/resources/site/freemarker/ndrs/components/homeland.ftl

```diff
@@ -0,0 +1,73 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+<#include "macro/metaTags.ftl">
+
+<#-- Add meta tags -->
+<@metaTags></@metaTags>
+
+<@hst.setBundle basename="homepage.website.labels"/>
+<@fmt.message key="data-and-information.title" var="dataAndInformationSectionTitle"/>
+
+<div>
+  <@hst.include ref="top"/>
+</div>
+
+<div class="page-block">
+    <div class="grid-wrapper grid-wrapper--article">
+
+        <#-- [FTL-BEGIN] 'Blog latest' section -->
```

Comment:
I am not used to the alternative `<#-- [FTL-BEGIN] 'Blog latest' section -->` FTL style. I have to trust they work correctly.